gcsfs/tests/conftest.py

```diff
@@ -63,7 +63,7 @@ def docker_gcs():
     container = "gcsfs_test"
     cmd = (
         "docker run -d -p 4443:4443 --name gcsfs_test fsouza/fake-gcs-server:latest -scheme "
-        "http -public-host http://localhost:4443 -external-url http://localhost:4443 "
+        "http -public-host 0.0.0.0:4443 -external-url http://localhost:4443 "
```

Comment:
C: it was needed to make signed URLs downloadable, otherwise it returns "Not found"

Reply:
OK. This probably depends on the specific network settings, but the changed version should always work.